nix/cardano-services/deployments/default.nix

```diff
@@ -383,7 +383,7 @@ in
             env.OVERRIDE_FUZZY_OPTIONS = "true";
           };
           handle-provider.enabled = true;
-          asset-provider.enabled = true;
```

Comment:
Is this superseded by https://github.com/input-output-hk/cardano-js-sdk/pull/1293 ?

Reply:
Thanks for the reminder, I will get that PR merged first and work as that as source of truth. Will get rid of this pr for now